stories/gender-by-country.md

```diff
@@ -10,6 +10,14 @@
 -->
 
 <figure style="text-align:center; margin: 40px 0;">
-<img src="data/by-culture.png" height="400px"></img>
-<figcaption> Lorem ipsum </figcaption>
+<!-- <img src="data/by-culture.png" height="400px"></img> -->
+<script
+    src="scripts/gender_by_country.js"
+    id="7e8b778e-0622-43d4-a67c-be22648ffaf7"
```

Comment:
We get a new `id` every time we redraw the plot, we need to code to put the tag in the `md` files instead rather than doing it by hand.